build.rs

```diff
@@ -64,6 +66,11 @@ fn cc_flags() -> Vec<&'static str> {
         ]);
     }
 
+    if target.contains("apple") || target.contains("freebsd") {
+        result.push("-stdlib=libc++");
+        result.push("-mmacosx-version-min=10.7");
```

Comment:
Yeah I've seen that we can do that but I would like to understand why this is even needed on Travis when I don't need it locally.

Btw this block is also executed when `target` contains "freebsd".